.storybook/preview.js

```diff
@@ -0,0 +1,6 @@
+import { action } from "@storybook/addon-actions";
+import { configure } from "@storybook/react";
+
+import "./stories.css";
+
+configure(require.context("../src", true, /\.stories\.mdx$/), module);

```

Comment:
this feels like a duplication of `stories: ['../src/**/*.stories.@(js|mdx)'], ` in main.js - do we need both? what is each for?

Reply:
You are right, this line is not needed. 
In the previous version we only had 1 file called config.js to configure everything, now we have main.js to indicate the location of the story files and the list of addons used. Preview.js configures the iframe that renders components, it is the file where we need to import global stylesheet for exemple.